usermods/usermod_v2_four_line_display_ALT/platformio_override.sample.ini

```diff
@@ -7,11 +7,12 @@ platform = ${esp32.platform}
 build_unflags = ${common.build_unflags}
 build_flags =
     ${common.build_flags_esp32}
-    -D USERMOD_FOUR_LINE_DISPLAY -D USE_ALT_DISPlAY
-    -D USERMOD_ROTARY_ENCODER_UI -D ENCODER_DT_PIN=18 -D ENCODER_CLK_PIN=5 -D ENCODER_SW_PIN=19
-upload_speed = 460800
+    -D USERMOD_FOUR_LINE_DISPLAY
+    -D FLD_TYPE=SH1106
+    -D I2CSCLPIN=27
+    -D I2CSDAPIN=26
+
 lib_deps =
     ${esp32.lib_deps}
-    U8g2@~2.34.4
+    U8g2
```

Comment:
Hello,
because the code is full of referencies to different versions on U8g2:
[here](https://github.com/Aircoookie/WLED/blob/2703c9899afd6a9c6f3f41eccce03b6b11208a0a/platformio.ini#L152)
[here](https://github.com/Aircoookie/WLED/blob/2703c9899afd6a9c6f3f41eccce03b6b11208a0a/platformio_override.sample.ini#L22)
[here](https://github.com/Aircoookie/WLED/blob/2703c9899afd6a9c6f3f41eccce03b6b11208a0a/platformio_override.sample.ini#L384)

I am using the latest version in my build, without issue.

Reply:
We always pin libraries to the specific version we tested with, to ensure that we get the same sources if we check out the code in the future.  Otherwise we end up with issues when our builds suddenly break -- and old builds won't compile! -- when upstream pushes new code.

Personally I have no objections to updating to a newer version, so long as other devs with compatible hardware can validate the change, but you must provide a specific version pin.

Reply:
Ok, I've fixed back.